inference-engine/tests/functional/shared_test_classes/src/read_ir/read_ir.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2019-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "functional_test_utils/core_config.hpp"
+#include "shared_test_classes/read_ir/read_ir.hpp"
+
+namespace LayerTestsDefinitions {
+
+static bool isRoiOperation(const std::shared_ptr<ngraph::Node>& op) {
+    return (ngraph::is_type<ngraph::op::v0::PSROIPooling>(op) ||
+            ngraph::is_type<ngraph::op::v0::ROIPooling>(op) ||
+            ngraph::is_type<ngraph::op::v3::ROIAlign>(op));
+}
+
+std::string ReadIRTest::getTestCaseName(const testing::TestParamInfo<std::tuple<std::string, std::string>>& obj) {
+    std::string pathToModel, deviceName;
+    std::tie(pathToModel, deviceName) = obj.param;
+
+    std::ostringstream result;
+    result << "ModelPath=" << pathToModel << "_";
+    result << "TargetDevice=" << deviceName << "_";
+    return result.str();
+}
+
+void ReadIRTest::SetUp() {
+    std::tie(pathToModel, targetDevice) = this->GetParam();
+}
+
+void ReadIRTest::LoadNetwork() {
+    cnnNetwork = getCore()->ReadNetwork(pathToModel);
+    function = cnnNetwork.getFunction();
```

Comment:
function creation should be moved to `SetUp` to keep all tests in shared_test_classes aligned. 
In current approach, this test can be considered as shared, since it's setup do nothing in fact and it can't be reused for another validation scenarios.
Also it will allow to avoid `LoadNetwork` overriding

Reply:
Agree